modules/test/playwright/pages/calendar-web/ModalRecurrencePage.ts

```diff
@@ -42,6 +51,24 @@ export class ModalRecurrencePage {
 		await this.doneButton.click();
 	}
 
+	async addRecurrenceUntilDate(daysFromNow: number) {
+		const nowDate = new Date();
+
+		const fiveDaysFromNowDate = new Date(
```

Comment:
:thinking: Considering that anyone can call `fillEventWithRecurrenceUntilDate` with any value they need, and it can be different from 5, why did you name this instance like this?